.storybook/Button/Button.stories.jsx

```diff
@@ -20,42 +21,34 @@ export const ButtonMultiple = () => (
     </>
 );
 
-ButtonNormal.story = {
-    parameters: {
-        zeplinLink:
-            "zpl://components?pid=5ecff3a2d8a8ab2a61937a66&coids=6050f561b398fd6ec62563ff",
-    },
+ButtonNormal.parameters = {
```

Comment:
[Hoisted CSF annotations](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#hoisted-csf-annotations) were introduced in Storybook 6.0